DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/storm_control/storm_control_utils.py

```diff
@@ -1,8 +1,34 @@
 import math
+import asyncio
 
 from dent_os_testbed.lib.devlink.devlink_port import DevlinkPort
 from dent_os_testbed.lib.tc.tc_filter import TcFilter
 
+from dent_os_testbed.test.test_suite.functional.devlink.devlink_utils import (
+    verify_devlink_cpu_traps_rate_avg, SCT_MAP
+)
+
+
+def get_streams(list_streams, prefixes):
+    filtered_streams = {}
+    for stream_name, stream in list_streams.items():
+        for prefix in prefixes:
+            if stream_name.startswith(prefix):
+                filtered_streams[stream_name] = stream
+                break
+    return filtered_streams
+
+
+async def verify_cpu_rate(dent_dev, deviation):
+    new_map = SCT_MAP.copy()
```

Comment:
This will create a shallow copy of SCT_MAP (not a deep copy) -> thus it will modify the original SCT_MAP. Please use copy.deepcopy() here.